### ISSUE TITLE
Set uart default baud rate to 74880 may make users confuse

### ISSUE THREAD

**Issue by tanhangbo** [espressif/esp8266-rtos-sample-code#6]
A lot of devices use 115200 and the USB-TTL I use with CH340 chip seems not support this a little strange baudrate. When I first play with the AI Thinker IDE (default with the uart example), I can only see some nonsense characters to the serialport(the ROM boot also prints some code I cannot ensure what it is) until I have found it uses the 74880 baudrate, it spent me a lot of time to check the  hardware&software&configration to make the helloworld work.

**Comment by GaryHendrick** [comment]
It is a silly idea to use 74880, a number of common serial communications applications do NOT support 115200 baud.

**Comment by de1m** [comment]
yes, this was also my [problem](https://github.com/espressif/ESP8266_RTOS_SDK/issues/296)

**Comment by FunMiles** [comment]
Is there a way to actually get 115200 bauds out of the board?
The sample code I use starts with:
`// Init UART0 and set baud rate to 115200 bps`
` uart_init_new();`

Obviously the comment is wrong. My monitor gets output when I set it to 74880, but I lose the beginning of the output while switching speed.
